app/services/people/finder.rb

```diff
@@ -19,14 +19,27 @@ def apply_filters(scope)
     end
 
     def apply_police_national_computer_filters(scope)
-      return unless filter_params.key?(:police_national_computer)
+      return unless filter_params.key?(:police_national_computer) || filter_params.key?(:nomis_offender_no)
 
       scope = scope.joins(:profiles)
-      scope.where('profiles.profile_identifiers @> ?', police_national_computer)
+
+      if filter_params.key?(:police_national_computer)
```

Comment:
It would have been nice to DRY up this class, especially as it now deals with 2 types of identifiers.

Reply:
yes, we could using `select`, `maps`, ... to DRY it,
but it would be less readable. This method is still quite small, I'd keep it as it is if you are happy.